### PR TITLE
Release 1.0.3

### PR DESCRIPTION
**Pull request by wagner-certat**
The changelog is already quite long, time for a new release!

There are currently open open PRs (#1160) and 11 open issues. Nothing critical, the PR can still be merged once reviewed.

Please don't merge yet, I'll add release dates once this passed the review process.

### CODE REVIEW

File: intelmq/etc/defaults.conf [hunk 1]
```diff
@@ -6,6 +6,7 @@
     "destination_pipeline_password": null,
     "destination_pipeline_port": 6379,
     "error_dump_message": true,
+    "error_forward_message": false,
```
Comment (SYNchroACK) [inline intelmq/etc/defaults.conf]: @wagner-certat i dont know if I added this by accident...

File: intelmq/etc/defaults.conf [hunk 2]
```diff
@@ -21,7 +22,7 @@
     "log_processed_messages_count": 500,
     "log_processed_messages_seconds": 900,
     "logging_handler": "file",
-    "logging_level": "INFO",
+    "logging_level": "DEBUG",
```
Comment (SYNchroACK) [inline intelmq/etc/defaults.conf]: also this one...

**Comment by SYNchroACK** [comment]
> There are currently open open PRs (#1160) and 11 open issues. Nothing critical, the PR can still be merged once reviewed.

@wagner-certat #1160 updated as you requested

File: intelmq/etc/defaults.conf (hunk 1 above)
Comment (wagner-certat) [reply intelmq/etc/defaults.conf]: Thanks for catching these!

**Pull request merged**